var/spack/repos/builtin/packages/py-tomopy/package.py

```diff
@@ -16,7 +16,9 @@ class PyTomopy(PythonPackage):
     git      = 'https://github.com/tomopy/tomopy.git'
 
     version('master', branch='master')
-    version('1.10.4', sha256='2f15edda11b4337a1a5560684fcf8f28a35c5035932b22b842ce728bd13fba01')
+    version('1.11.0', sha256='4e5691c2b083753692ba4376ce301578037071c83fc61a6ae9e5bc9e6fcd3d1f')
+    # Never was an official 1.10.4 release, checksum doesn't match for some reason
+    version('1.10.4', sha256='2f15edda11b4337a1a5560684fcf8f28a35c5035932b22b842ce728bd13fba01', deprecated=True)
```

Comment:
You could (if you want) remove 1.10.4: I think I only added it because building another package required a newer version.

Likely I should have only added the version and set the default to 1.0.0.

Reply:
I'll deprecate it for now and remove it in the next release.